core/src/test/scala/unit/kafka/server/epoch/EpochDrivenReplicationProtocolAcceptanceTest.scala

```diff
@@ -53,17 +58,26 @@ class EpochDrivenReplicationProtocolAcceptanceTest extends QuorumTestHarness wit
   val topic = "topic1"
   val msg = new Array[Byte](1000)
   val msgBigger = new Array[Byte](10000)
-  var brokers: Seq[KafkaServer] = _
+  var brokers: Seq[KafkaBroker] = _
   var producer: KafkaProducer[Array[Byte], Array[Byte]] = _
   var consumer: Consumer[Array[Byte], Array[Byte]] = _
 
+  private def securityProtocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
+  private def listenerName: ListenerName = ListenerName.forSecurityProtocol(securityProtocol)
+
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
+    if (TestInfoUtils.isKRaft(testInfo) && metadataVersion.isLessThan(IBP_3_3_IV0)) {
```

Comment:
Can you explain why we need this change? `metadataVersion` is hard coded to `MetadataVersion.latestTesting`, how can it be less than `IBP_3_3_IV0`?

Reply:
Oh thank you! It seems unnecessary. I'll edit it